setup.py

```diff
@@ -1,18 +1,43 @@
 #!/usr/bin/python3
 import setuptools
+import sys
 import PyInstaller.__main__ as pim
 
-pim.run([
-	"--name=DodgeToDrums",
-	"--onefile",
-	"--windowed",
-	"--clean",
-	"--exclude-module=tkinter",
-	"--exclude-module=PyQt4",
-	"--exclude-module=PyQt5",
-	"--exclude-module=numpy",
-	"--hidden-import=packaging.requirements",
-#	"--icon=icon.ico",
-#	"--add-data=./data/",
-	"main.py"
-	])
+DEFAULT_PROFILE = "dist"
+
+if len(sys.argv) > 0:
```

Comment:
sys.argv usually not only contains arguments, but also the script location, therefore it should be `…>1:`
also, the arguments should be parsed completely differently: first, they should get ordered into shorts, longs and absolutes (-s --long absolute), then every short, long and absolute should be checked and processed (with according constants being set), if any option is non-existent or more than one absolute is present, throw an error according to that. I know that's a bit much, but argument handling has to be done right.

Reply:
Yeah, I thought I had put 1 there sry

Reply:
And yes it is to much, but thankfully python has a standard libary for argument handling